packages/p2p/src/components/app-content.jsx

```diff
@@ -32,6 +33,10 @@ const AppContent = () => {
         return <Verification should_wrap />;
     }
 
+    if (buy_sell_store?.show_advertiser_page && !buy_sell_store.should_show_verification) {
+        return <AdvertiserPage />;
+    }
+
```

Comment:
I spoke to Yunus about the tabs and we want to change it in the next release. You can leave this as it is for now and add a TODO on top to discuss with design team and change it

Reply:
Got it!